packages/editor/src/components/writing-flow/index.js

```diff
@@ -262,7 +263,7 @@ class WritingFlow extends Component {
 			return;
 		}
 
-		// Abort if navigation has already been handled (e.g. TinyMCE inline
+		// Abort if navigation has already been handled (e.g. RichText inline
```

Comment:
We changed the name, but... do the rich text boundaries even cause `preventDefault` to be called? This seems like it was oriented to very specifically handle a TinyMCE behavior which may not apply any longer.

The same applies for the previous changes around handling Cmd+A . This would have been handled internally by TinyMCE, which as best I can tell is what prompted the workaround to be implemented.

Could either or both of these be removed now?

Reply:
Yes, we're still prevent defaulting on left/right arrow keys. I quickly tested if it would work without the check, but it doesn't. With cmd+a it's actually not TinyMCE but the browser.

Reply:
https://github.com/WordPress/gutenberg/pull/13697/files#diff-e373e566d15f8f4196eb0b3998adbdd7R660

Reply:
This makes sense to me, because when you're at a horizontal edge of the DOM, it doesn't necessarily mean that you're at the edge of the boundary. E.g. `<p><strong>|text<strong><p>`: | is not at the edge, but it is at the edge of the DOM. That's why we have to look at `preventDefault`.

Reply:
Okay, I'm convinced 😃